docs/dev/building.md

```diff
@@ -22,37 +22,44 @@ git clone https://github.com/scylladb/scylla
 cd ./scylla
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py --mode=<mode>
-./tools/toolchain/dbuild ninja
+./tools/toolchain/dbuild cmake --build build
 ```
 
 where `mode` is `dev` for development builds, `release` for release builds, and `debug` for debug builds.
```

Comment:
This line is out of date and must be fixed. The names are no different.

Reply:
i think this line is not outdated. `dev` , `release` and `debug` are still accepted by `configure.py`.